spec/unit/actions/v3/service_instance_delete_spec.rb

```diff
@@ -1,4 +1,4 @@
-require 'db_spec_helper'
+require 'spec_helper'
```

Comment:
I couldn't run this test file locally until I switched the import to `spec_helper`.